nextcloudappstore/core/models.py

```diff
@@ -118,6 +159,12 @@ def can_delete(self, user: User) -> bool:
     def __str__(self) -> str:
         return '%s %s' % (self.app, self.version)
 
+    def is_compatible(self, platform_version):
+        platform_version = Version(platform_version)
+        if platform_version in Spec(self.platform_version_spec):
```

Comment:
What about simply doing: return platform_version in Spec(self.platform_version_spec)